server/src/main/java/edu/mit/csail/db/ml/util/ContextFactory.java

```diff
@@ -9,11 +9,21 @@
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.SQLException;
+import java.util.Optional;
+
 
 /**
  * This class contains logic for connecting to the database.
  */
 public class ContextFactory {
+ static {
```

Comment:
This static initializer isn't needed in this PR (this is to workaround an issue bundling into a fat jar).

Reply:
removed the static initializer